src/lib/wicked/connections.js

```diff
@@ -121,7 +122,9 @@ const createConnection = (config) => {
     const mtu = config?.link?.mtu ? parseInt(config.link.mtu) : undefined;
     const usedBy = config?.link?.master;
 
-    return model.createConnection({
+    console.log(config);
```

Comment:
`console.log` left over here.

Reply:
Sorry, it was already fixed but the push was not forced.